index.html

```diff
@@ -746,6 +747,11 @@ <h2><dfn>readable</dfn> attribute</h2>
                     [=ReadableStream/error=] on [=this=].{{SerialPort/[[readable]]}} with
                     an "{{UnknownError}}" {{DOMException}} and invoke the steps
                     to [=handle closing the readable stream=].
+                  <li>
+                    If [=this=].{{SerialPort/[[state]]}} becomes `"forgotten"`, invoke
+                    [=ReadableStream/error=] on [=this=].{{SerialPort/[[readable]]}} with
+                    an "{{NetworkError}}" {{DOMException}} and invoke the steps
+                    to [=handle closing the readable stream=].
```

Comment:
This has to happen before the read completes, so I think we need a note on the "Invoke the operating system" step which says to treat becoming forgotten as if the port was disconnected.

Reply:
Done